### PR TITLE
Rl progress bar

### PR DESCRIPTION
**Pull request by lluneva**
ProgressBar for quiz completion.
STATIC progress inputs for now.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,13 +1,18 @@
 import React, { Component } from "react";
 import "./App.css";
 
+// import Login  from  "./components/Login/Login";
```
Comment (rokija) [inline src/App.js]: remove commented out code

File: src/App.js [hunk 2]
```diff
@@ -1,13 +1,18 @@
...
+import ProgressBar from "./components/ProgressBar/ProgressBar";
+
 class App extends Component {
   render() {
-    return <div className="App">
-   react App
-    </div>;
+    return (
+      <div className="App">
+        <ProgressBar currQuestion={17} totalQuestions={20}/>
```
Comment (rokija) [inline src/App.js]: rename to currentQuestion

File: src/components/ProgressBar/ProgressBar.css [hunk 3]
```diff
@@ -0,0 +1,14 @@
+.ProgressBar{
+    width: 350px
```
Comment (rokija) [inline src/components/ProgressBar/ProgressBar.css]: missing semicolon

File: src/components/ProgressBar/ProgressBar.css [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+.progress {
+    width: 350px;
+    border-radius: 10px;
+    height: 50px;
+}
+
+.ProgressBar__text {
+  font-size: 20px;
+
```
Comment (rokija) [inline src/components/ProgressBar/ProgressBar.css]: remove empty line

**Review by rokija — APPROVED** [review]

**Review by janisknets — APPROVED** [review]

**Pull request merged**